Libraries/Components/Touchable/TouchableWithoutFeedback.d.ts

```diff
@@ -58,7 +58,7 @@ export interface TouchableWithoutFeedbackProps
   /**
```

Comment:
It seems like you forgot to update the `touchSoundDisabled?` as well. Was it intentional?

Reply:
Ops, sorry my bad. I'll update it right now

Reply:
done